### PR TITLE
specs: special namespaces

### PR DESCRIPTION
**Pull request by jakubuid**

### CODE REVIEW

**Review by crypblizz8 — APPROVED** [review]
LGTM

File: docs/specs/clients/sign/data-structures.md [hunk 1]
```diff
@@ -162,3 +174,24 @@ Verify Context is appended to Session Proposals and Session Requests to provide
   }
 }
 ```
+
+## Namespace Config
+Namespace Config is set withing the setNamespaceConfig method that is a part of the Sign client. It defines Special and Supported Namespaces and must be called after initialization of the SDK and before approve session method.
+```jsonc
```
Comment (Elyniss) [inline docs/specs/clients/sign/data-structures.md]:
```suggestion
Namespace Config is set within the setNamespaceConfig method that is a part of the Sign client. It defines Special and Supported Namespaces and must be called after initialization of the SDK and before approve session method.
```jsonc
```

**Review by Elyniss — APPROVED** [review]

File: docs/specs/meta-clients/web3wallet/sdk-api.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ class Web3Wallet {
   
   // establish pairing from URI (BOTH)
   public abstract pair(params: { uri: string }): Promise<void>;
+
+  // for responder to set supported and special namespaces
+  public abstract setNamespaceConfig(params: { namespaceConfig: NamespaceConfig }): Promise<void>;
```
Comment (ganchoradkov) [inline docs/specs/meta-clients/web3wallet/sdk-api.md]:
```suggestion
  public abstract setNamespaceConfig(params: NamespaceConfig): Promise<void>;
```
this should be `params: NamespaceConfig` and params itself containing `supportedNamespaces & specialNamespaces` parameters

File: docs/specs/clients/sign/client-api.md [hunk 3]
```diff
@@ -16,10 +16,15 @@ abstract class Client {
     pairingTopic: string;
   }): Promise<Sequence>;
 
+  // for responder to set supported and special namespaces
+  public abstract setNamespaceConfig(params: {
```
Comment (ganchoradkov) [inline docs/specs/clients/sign/client-api.md]:
```suggestion
  public abstract setNamespaceConfig(params: NamespaceConfig) : Promise<void>;
```

File: docs/specs/clients/sign/client-api.md [hunk 4]
```diff
@@ -16,10 +16,15 @@ abstract class Client {
...
+     namespaceConfig: NamespaceConfig
```
Comment (ganchoradkov) [inline docs/specs/clients/sign/client-api.md]:
```suggestion
```

File: docs/specs/clients/sign/client-api.md [hunk 5]
```diff
@@ -16,10 +16,15 @@ abstract class Client {
...
+  }): Promise<void>;
```
Comment (ganchoradkov) [inline docs/specs/clients/sign/client-api.md]:
```suggestion
```

**Review by ganchoradkov — APPROVED** [review]
💯  small update requested regarding the `setNamespaceConfig ` params

**Pull request merged**